src/main/java/com/github/fukkitmc/fukkit/mixin/net/minecraft/block/TripwireBlockMixin.java

```diff
@@ -14,12 +14,14 @@
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+import java.util.Iterator;
 import java.util.List;
 
 @Mixin (TripwireBlock.class)
 public class TripwireBlockMixin {
 	@Inject (method = "updatePowered", at = @At (value = "JUMP", ordinal = 1, shift = At.Shift.AFTER), locals = LocalCapture.PRINT, cancellable = true)
-	private void fukkit_interactEvent(World world, BlockPos pos, CallbackInfo ci, BlockState state, boolean powered, boolean shouldPower, List<? extends Entity> list) {
+	private void fukkit_interactEvent(World world, BlockPos pos, CallbackInfo ci, BlockState state, boolean powered, boolean shouldPower, List<? extends Entity> list, Iterator<?> var7) {
```

Comment:
Is this iterator on the end even used within the code? If it isn't, you can remove the iterator since you only need to specify locals up to the last param you need

Reply:
No, I forgot it. I will remove it later.